src/components/Main-nav/MainNav.js

```diff
@@ -0,0 +1,66 @@
+import template from './MainNav.pug'
+import data from '../../../data/mainNav.json'
+import './MainNav.scss'
+
+export class MainNav {
+  constructor (node) {
+    this.node = node
+    this.elements = {}
+    this.insertHTML(this.node)
+    this.setContainer()
+    this.setEventHamburger()
+    this.setEventlist()
+    this.currentListButton = null
+  }
+
+  insertHTML (node) {
+    node.innerHTML = template(data)
+  }
+
+  setContainer () {
+    this.elements.navBar = this.node.querySelector('.main-menu__nav-bar')
+    this.elements.list = this.node.querySelector('.main-menu__list')
+    this.elements.listItems = this.elements.list.querySelectorAll('.main-menu__list-item')
+    this.elements.itemButtons = this.elements.list.querySelectorAll('.main-menu__list-item-button')
+    this.elements.listLinks = this.elements.list.querySelectorAll('.main-menu__list__link')
+  }
+
+  setEventHamburger () {
+    this.elements.hamburger = this.elements.navBar.querySelector('.main-menu__hamburger')
+    this.elements.hamburger.addEventListener('click', () => {
+      this.elements.hamburger.classList.toggle('main-menu__hamburger--clicked')
```

Comment:
`main-menu__hamburger--clicked` should be a constant.

Reply:
I don´t understand. Do you say that I change `main-menu__hamburger--clicked` for `main-menu__hamburger-clicked` ?